ast.go

```diff
@@ -885,6 +923,9 @@ type RevokeStatement struct {
 
 // String returns a string representation of the revoke statement.
 func (s *RevokeStatement) String() string {
+	if s == nil {
+		return ""
```

Comment:
Need to check if `s.Privilege` is nil if it is a pointer type.

Reply:
it is an int type